src/Microsoft.Azure.SignalR.Protocols/MulticastDataMessage.cs

```diff
@@ -141,6 +141,16 @@ public class GroupBroadcastDataMessage : MulticastDataMessage
         /// </summary>
         public IReadOnlyList<string> ExcludedList { get; set; }
 
+        /// <summary>
+        /// Gets or sets the list of excluded user Ids.
+        /// </summary>
+        public IReadOnlyList<string> ExcludedUserList { get; set; }
+
+        /// <summary>
+        /// Gets or sets the user ID of the sender
+        /// </summary>
+        public string SenderId { get; set; }
```

Comment:
CallerId?

Reply:
Since it is used in `SendToGroup`, does `sender` make more sense?

Reply:
caller sounds more general and can be applied to other messages

Reply:
since this is a `message`, I would prefer using `sender`.